dissect/target/plugins/browsers/iexplore.py

```diff
@@ -29,10 +29,11 @@ def find_containers(self, name: str) -> table.Table:
         try:
             for container_record in self.db.table("Containers").records():
                 record_name = container_record.get("Name")
-                record_name = record_name.rstrip("\00").lower()
-                if record_name == name.lower():
-                    container_id = container_record.get("ContainerId")
-                    yield self.db.table(f"Container_{container_id}")
+                if record_name is not None:
```

Comment:
```suggestion
                if record_name:
```
Could also make it a walrus operator.